db/migrate/20221031193754_devise_create_users.rb

```diff
@@ -4,6 +4,9 @@ class DeviseCreateUsers < ActiveRecord::Migration[5.2]
   def change
     create_table :users do |t|
       ## Database authenticatable
+      t.string :first_name, null: false
+      t.string :last_name, null: false
+      t.string :role, null: false, default: "member"
```

Comment:
Not something that needs to be changed right now, but generally when setting up a `role` we're better off using an `enum` data type than just a string. Having a string data type means we could have users with roles of "member", "admin", "cat", "mmber" (spelt incorrectly) leaving the implementation fragile. Whereas an enum allows us to designate the options that are allowed!